### PR TITLE
Review modal work

### PR DESCRIPTION
**Pull request by CatMo9**
Today I worked on the thumbnail images and creating a model to display them full sized when clicked. Worked on figuring out the modal for the New Review and created placeholders for the data to go back and fill in later. 
I also finished up the helpful and report links for each of the reviews. These are complete with axios requests to the API for updating. 
I had to refactor some of my functions to be able to conditionally render a longer body and a link to click if there is more body text to show.

### CODE REVIEW

File: client/src/components/reviews/apiExample.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ let data = {
         "summary": "ReviewTitle",
         "recommend": true,
         "response": null,
-        "body": "Review Body For The Review That We Are Testing This Review Text Must Meet A Minumum Number Of Characters To Work But I Dont Know Why I Did A Camel Case Sentence Thats Just Weird",
+        "body": "Review Body For The Review That We Are Testing This Review Text Must Meet A Minumum Number Of Characters To Work But I Dont Know Why I Did A Camel Case Sentence Thats Just Weird. Why can I simply look out the window for hours on end. I love my life, I love the weather, I love nature. Please let this sentence be long enough now. I do not care about this product really. Shhhhhhh, it is a secret!",
```
Comment (Chrisgood2go) [inline client/src/components/reviews/apiExample.js:13]: made me smile.

File: client/src/components/reviews/reviewListComponents/IndividualTile.jsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+import API_KEY from '../../../../../config.js'
+const {Checkmark} = require('react-checkmark');
+
+
+// FORMATING FUNCTIONS
+
+export const makeDate = (dateData) => {
+  // without these regular expressions at the end of the dateData, the date shows up as off by a day. Adding these filters fixed that issue
+  // this line writes a date stamp into human legible dates ex: Mon Jul 12 2019.
+  let formatedDate = new Date(dateData.replace(/-/g, '\/').replace(/T.+/, '')).toDateString();
```
Comment (Chrisgood2go) [inline client/src/components/reviews/reviewListComponents/IndividualTile.jsx:14]: wow, using regex. You're a lady and scholar Cathy!

File: client/src/components/reviews/reviewListComponents/IndividualTile.jsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+  let [showLongBody, setShowLongBody] = useState(false);
+  const changeBody = () => {
+    notYetDisplayed = false;
+    setShowLongBody(true);
+  }
+
+  let recommend = makeRecommendation(review.recommend);
+  let date = makeDate(review.date);
+  let summary = makeSummary(review.summary);
+  let body = makeBody(review.body)
```
Comment (Chrisgood2go) [inline client/src/components/reviews/reviewListComponents/IndividualTile.jsx:148]: ;

File: client/src/components/reviews/reviewListComponents/ReviewTiles.jsx [hunk 4]
```diff
@@ -1,86 +1,9 @@
 import React from 'react';
-import {ReviewTileContainer, RecommendationContainer, SellersResponse} from '../sharedStyles/sharedStyledComponents';
-const {Checkmark} = require('react-checkmark');
+import {IndividualTile} from './IndividualTile.jsx';
```
Comment (Chrisgood2go) [inline client/src/components/reviews/reviewListComponents/ReviewTiles.jsx:2]: this was from nick adams' modal right?

**Review by Chrisgood2go — APPROVED** [review]
I put one comment for what I thought was a missing semicolon.

File: client/src/components/reviews/reviewListComponents/ReviewTiles.jsx (hunk 4 above)
Comment (CatMo9) [reply client/src/components/reviews/reviewListComponents/ReviewTiles.jsx:2]: Actually this was me moving my files from ReviewTiles into IndividualTile. I was mapping twice in the one file and it was causing issues, so I had to move them so there was only one map function per file.

**Pull request merged**